### ISSUE TITLE
Fix: Layout issues. 

### ISSUE THREAD

**Issue by tejasbenibagde** [Luson045/medi-connect#199]
# Fix: Layout Issues

Hello @Luson045,

I went through your site and identified some layout inconsistencies that I'd like to address. Below are the specific issues:

## Screenshots

1. As you can see in this image, the sizes of the cards are not consistent. (some cards are large while some are small)
   
   ![layout issue](https://github.com/user-attachments/assets/ebd84961-a183-4742-8c51-ea1f19a4f83f)

2. In this screenshot, the cards are taking up the full screen width, even though they don't require that much space.
   
   ![layout issue2](https://github.com/user-attachments/assets/90ecae49-fcf0-444a-a9e3-4b8c617480e5)

I believe fixing these issues will improve the user experience. Could you please assign this task to me?

### Related Tags
- gssoc-ext
- hactoberfest

Also, please add the appropriate GSSOC level and assign me to this task.

Thank you!

**Comment by Luson045** [comment]
@tejasbenibagde  work is being done on tht

**Comment by tejasbenibagde** [comment]
I actually fixed this issue in my last commit.

if you don't mind can I create a pull request now ?

**Comment by Luson045** [comment]
Share the screenshots of the changes, I've paused deployment of PRs

**Comment by tejasbenibagde** [comment]
Ok give me a second I'll share the screenshot.

**Comment by tejasbenibagde** [comment]
@Luson045 here are some screenshots you have requested. 

1. here I have made the changes making sure that each card on a same row has the same dimension

![fix1](https://github.com/user-attachments/assets/171e8f09-a28e-49dd-9610-3de1d58934b0)

2. here I have changed the layouts of card so that they take only the space they need. 

![fix2](https://github.com/user-attachments/assets/f02664b1-cfa9-4180-bf9b-029aeb578883)
